docs/DeveloperGuide.adoc

```diff
@@ -279,51 +279,163 @@ Refer to the guide <<DevOps#, here>>.
+
+|`* *` | user | set a recurring reminder for contacts I loaned from | know when to pay them back
+
+|`* *` | user | add profile picture to a contact | know how they look like.
+
+|`* *` | user | export my data and import it from a different application| can sync my debt through multiple devices.
+
+|`*` | user| set up automated periodic notifications (emails, etc.) to people who owe me money | remind them to pay me back.
+
+|`*` | user| keep track of objects that I lent and borrowed from other people | remember to return them.
+
+|`*` | user| specify conditions on the loans | assure the loaned items are returned in a specified condition.
+
+|`*` | user| export the data to other format (.ics etc.)i | export the data to other applications.
+
+|`*` | traveller| create a common pool of funds to spend for large group activities| simplify tracking of common expenses
+
+|`*` | user| extract information from a picture of a bill| save the trouble of entering expense details myself.
+
+|=======================================================================
 
 [appendix]
 == Use Cases
 
-(For all use cases below, the *System* is the `AddressBook` and the *Actor* is the `user`, unless specified otherwise)
+(For all use cases below, the *System* is the `SplitWiser` and the *Actor* is the `user`, unless specified otherwise)
+
+[discrete]
+=== Use case: List contacts
+
+*MSS*
+
+1. User requests to list contacts
+2. SplitWiser shows a list of contacts
++
+Use case ends.
+
+*Extensions*
+
+[none]
+* 1a. The list of contacts is empty
++
+Use case ends.
+
+[discrete]
+=== Use case: List activites
+
+*MSS*
+
+1. User requests to list activities
+2. SplitWiser shows a list of activities
++
+Use case ends.
+
+*Extensions*
+
+[none]
+* 1a. The list of activities is empty
++
+Use case ends.
 
 [discrete]
-=== Use case: Delete person
+=== Use case: View an activity
 
 *MSS*
 
-1.  User requests to list persons
-2.  AddressBook shows a list of persons
-3.  User requests to delete a specific person in the list
-4.  AddressBook deletes the person
+1. User requests to view an activity with a specific activity ID
+2. SplitWiser shows details of the specified activity
++
+Use case ends.
+
+*Extensions*
+[none]
+* 1a. No activity exists with the specified activity ID.
++
+[none]
+** 1a1. SplitWiser shows an error message.
++
+Use case ends.
+
+[discrete]
+=== Use case: Create an activity
+
+*MSS*
+
+1. User requests to create an activity with a given title and participant(s)
+2. SplitWiser creates the activity with the supplied title and no expenses
+3. SplitWiser adds the user and the supplied contact(s) to the activity
+4. SplitWiser shows details of the newly created activity
++
+Use case ends.
+
+*Extensions*
+[none]
+* 1a. User specifies one or more participants that are not found in the list of contacts
++
+[none]
+** 1a1. SplitWiser prompts the user to [.underline]#create new contacts# for each of the missing participants
```

Comment:
What if the User cancels during the step "creating new contacts"? Does this current use case end?